nri/index.html

```diff
@@ -251,10 +264,10 @@ <h3>Follow Us</h3>
   <footer>
     <ul class="footer-list">
       <li><a href="">免責条項</a></li>
```

Comment:
hrefがからなので#をつけておくと良いと思います。（他と合わせる）。
```suggestion
      <li><a href="#">免責条項</a></li>
```